tests/DoctrineDBALAdapterTest.php

```diff
@@ -14,14 +15,36 @@
  */
 class DoctrineDBALAdapterTest extends FilesystemAdapterTestCase
 {
-    protected static function createFilesystemAdapter(): FilesystemAdapter
+    protected static function createFilesystemAdapter(string $table = 'flysystem_files', ?string $prefix = ''): FilesystemAdapter
     {
         $connection = DriverManager::getConnection([
             'url' => 'sqlite:///:memory:',
         ]);
 
         $connection->executeStatement((string) file_get_contents(dirname(__DIR__).'/schema/sqlite.sql'));
 
-        return new DoctrineDBALAdapter($connection);
+        return new DoctrineDBALAdapter(connection: $connection, table: $table, prefix: $prefix);
+    }
+
+    public function testReadWriteWithPrefix(): void
```

Comment:
isn't this covered by the Flysystem tests?
Do we need a separate test?

Reply:
I created another test class that create adapter with prefix. This was missing.